File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Customer_Update.ascx.cs
using NotificationSystem.NotificationSystem.Data.Classes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace NotificationSystem.NotificationSystem.Web.Controls_Search
{
	public partial class ctrSearch_Customer_Update : System.Web.UI.UserControl
	{
		protected void Page_Load(object sender, EventArgs e)
		{
			clsNotificationSystem theNotificationSystem = new clsNotificationSystem();

			if (!(Page.IsPostBack))
			{
				cmbCustomer.DataSource = theNotificationSystem.GetCustomer();
				cmbCustomer.DataTextField = "CustomerID"; cmbCustomer.DataValueField = "FirstN";
				cmbCustomer.DataBind();
			}
		}

		public void ClearControl()
		{
			this.cmbCustomer.Value = string.Empty; cmbCustomer.Text = "--";
		}
	}

}

File: NotificationSystem/NotificationSystem.Web/Controls_Search/ctrSearch_Customer_Update.ascx.vb
Imports System
Imports System.Web.UI

Namespace NotificationSystem.NotificationSystem.Web.Controls_Search
    Public Partial Class ctrSearch_Customer_Update
        Inherits UserControl
        Protected Sub Page_Load(sender As Object, e As EventArgs)
            Dim theNotificationSystem As NotificationSystem.NotificationSystem.Data.Classes.clsNotificationSystem = New NotificationSystem.NotificationSystem.Data.Classes.clsNotificationSystem()

            If Not MyBase.Page.IsPostBack Then
                cmbCustomer.DataSource = theNotificationSystem.GetCustomers()
                cmbCustomer.DataTextField = "CustomerID"
                cmbCustomer.DataValueField = "FirstN"
                cmbCustomer.DataBind()
            End If
        End Sub

        Public Sub ClearControl()
            cmbCustomer.Value = String.Empty
            cmbCustomer.Text = "--"
        End Sub
    End Class

End Namespace
